crates/turborepo-api-client/src/lib.rs

```diff
@@ -264,31 +267,33 @@ impl APIClient {
         false
     }
 
-    pub fn new(base_url: impl AsRef<str>, timeout: Option<u64>) -> Result<Self> {
+    pub fn new(
+        base_url: impl AsRef<str>,
+        timeout: Option<u64>,
+        version: &'static str,
+    ) -> Result<Self> {
         let client = match timeout {
             Some(timeout) => reqwest::Client::builder()
                 .timeout(std::time::Duration::from_secs(timeout))
                 .build()?,
             None => reqwest::Client::builder().build()?,
         };
 
+        let user_agent = format!(
```

Comment:
We make user_agent a field in APIClient so that turborepo-lib can pass the version info instead of having turborepo-api-client have a dependency on get_version. That way we don't have to share get_version between turborepo-api-client and turborepo-lib.

Reply:
I think that is a good choice.